src/components/Header.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Kindly remove this eslint disable rule and include prop type definition for the `search` prop.

Reply:
Thank you @mikenath223 
I have resolved this issue